application.go

```diff
@@ -69,6 +88,7 @@ type Application struct {
 	AcceptedResourceRoles []string            `json:"acceptedResourceRoles,omitempty"`
 	LastTaskFailure       *LastTaskFailure    `json:"lastTaskFailure,omitempty"`
 	Fetch                 []Fetch             `json:"fetch"`
+	IpAddress							*IpAddress					`json:"ipAddress,omitempty"`
```

Comment:
`IpAddress` -> `IPAddress`

The formatting looks off. Did you run `go fmt` on your code?